content/en/real_user_monitoring/mobile_and_tv_monitoring/unity/data_collected.md

```diff
@@ -1,11 +1,9 @@
 ---
-title: RUM Unity Data Collected
```

Comment:
I think Data Collected still needs to be added under Unity in the nav, see [the staging page](https://docs-staging.datadoghq.com/rtrieu/rum-mobile-sdk-docs-restructure/real_user_monitoring/mobile_and_tv_monitoring/unity/data_collected/)